schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animal (
```

Comment:
- [ ] Kindly make sure that the table's name is `animals` not animal, change it.

Reply:
- Please, kindly make sure the table name is animals and not animal as requested by the previous reviewer.